### DISCUSSION TITLE
Replace main production branch on free tier?

### DISCUSSION THREAD

**Discussion by djordje-st** [planetscale/discussion#286] [Q&A] [answered]
As the title says I'm trying to replace the main production branch with a backup I made.

The dashboard is trying to make the backup branch as a production branch in addition to the main one which is where the issue occurs.

**Comment by orware** [comment, marked as answer]
@EvonuX,

This is a known issue that we are working to make better for our users.

In the meantime, please reach out to us via https://support.planetscale.com/ and you can provide the extra detail about your database in the request and we have some workarounds available for you we can provide to help you get that backup restored.
